src/common/actions/queries/findProjectsForCycle.js

```diff
@@ -16,6 +16,10 @@ export default function findProjectsForCycle({cycleNumber} = {}) {
             id
             cycleNumber
             state
+            startOfWeek {
+              unixTimestamp
+              mmddyy
+            }
```

Comment:
As noted, let's keep this simple in the API and just specify a `weekStartedAt` field in the `Project` graphql schema.